tests/acm-normal.expected.yaml

```diff
@@ -527,6 +527,7 @@ spec:
                       valueFiles:
                       - "/values-global.yaml"
                       - "/values-acm-edge.yaml"
+                      - "/values-.yaml"
```

Comment:
Should there be a default?

Reply:
I should actually wrap all those "/values-<ocpversion>.yaml" in some { if .global.clusterVersion }, to be on the safer side in case something goes wrong in fetching the clusterVersion